tests/baselines/reference/derivedClassSuperCallsInNonConstructorMembers.errors.txt

```diff
@@ -49,97 +24,47 @@ tests/cases/conformance/classes/constructorDeclarations/superCalls/derivedClassS
            ~~~~~
 !!! error TS2304: Cannot find name 'super'.
                 ~
-!!! error TS1005: ';' expected.
+!!! error TS1440: Function call not allowed at this location.

```

Comment:
This is definitely better, though I think the messages that say what is allowed, instead of what is not allowed, are best.

Reply:
@andrewbranch agreed. Do you have an idea for a better error message I could use in this generic spot?

Reply:
We need to parse a type annotation here, right? I would have expected something like “Type annotation expected” or “Function call is not a type annotation,” but I don’t know how easy it is to apply that in general.

Reply:
Nice, yeah, `parseSemicolonAfterPropertyName` is only called by `parsePropertyDeclaration` right now so we can make that assumption. I like `Function call is not a type annotation.` for being specific to this issue. Thanks!